tests/.wakeroot

```diff
@@ -1 +1 @@
-{}
+{"log_header":"", "log_header_source_width":0}
```

Comment:
I can't remember what we said  before but if these can just be `{}` I'd prefer that. If they can't then feel free to leave it

Reply:
It adds log headers if I don't add those things and consequently I'd have to go change all of the stdout/stderr parts if I didn't add this.